### DISCUSSION TITLE
GitHub Pages still redirect wrongly after removing custom domain

### DISCUSSION THREAD

**Discussion by skadomsky** [community/community#35295] [Other Feature Feedback, Questions, & Ideas] [answered]
I entered a incorrect domain there by mistake, then i **remove that domain later**. But GitHub pages still redirect to [that non-existing site](https://ska.eu.org), not even works after i redeployed it.
Here is [my current site](https://skadomsky.github.io). What should i do to stop it from redirecting? Thanks for answering!

**Comment by skadomsky** [comment, marked as answer]
Thanks to all. Problem solved :-)
Firefox Mobile's fault.

**Comment by felipepitol** [comment]
I'm having the same issue. I do not have custom domains. I'm trying to use the original github page for my project. But still redirecting to a custom domain.

**Reply by skadomsky** [reply]
I cleared browser cache and it all went on well.
Maybe try another browser?

**Comment by smartnftcoder1003** [comment]
Hello, @everyone.
I have issue in github pages. my username is smartnftcoder1003.
So repo name was smartnftcoder1003.github.io.
First I set custom domain, and then I removed it.
But still site redirect the custom domain.
And I cleard browser cache.
Who can solve this problem?
Thanks for your time.

**Reply by skadomsky** [reply]
I reached your site. It shows 'site not found' message by GitHub Pages.
The problem probably is, you forget to delete **DNS record** which you have set for redirecting to GitHub Pages.
I have faced similar situation before, simply delete that record from resolve host just works.

**Reply by New-Zealandball** [reply]
or is the file index.html

**Comment by AssahBismarkabah** [comment]
Greetings everyone, 
I am encountering a similar issue with a custom domain provisioned through AWS. Despite having disabled the DNS, the redirect continues to resolve to the original domain. @skadomsky, do you have any updates regarding this matter?

**Comment by MorningMC** [comment]
I am facing the same issue. I have tried flushing dns but it does not work.
Anyone helps?
Browser: Edge Domain: morningmc.github.io Redirects to: morning.mc

**Reply by MorningMC** [reply]
Problem solved. You can go to Settings -> Privacy -> Delete browsing data -> Clear browsing data now and click Choose what to clear.
Select Cached images and files in the popup and click Clear now.

**Reply by AssahBismarkabah** [reply]
> Problem solved. You can go to Settings -> Privacy -> Delete browsing data -> Clear browsing data now and click Choose what to clear. Select Cached images and files in the popup and click Clear now.

This works well for me, and the propagation is fast. However, I frequently experience downtime when trying to access the site.

**Comment by myudak** [comment]
BRO I STILL HAVE IT  BUGG MOTHERFUCKER GITHUB SUCKS BALLS
